pytorch_lightning/profiler/xla.py

```diff
@@ -0,0 +1,110 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+XLA Profiler will help you debug and optimize training workload performance
+for your models using Cloud TPU performance tools.
+
+Manual capture via TensorBoard
```

Comment:
How do these look in generated docs? Should it be something like this for headings:
```suggestion
Manual capture via TensorBoard
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```